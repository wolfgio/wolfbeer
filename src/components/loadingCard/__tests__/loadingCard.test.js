import React from 'react';
import { create, act } from 'react-test-renderer';

import LoadingCard from '../loadingCard';

describe('<LoadingCard />', () => {
  it('should render without error', () => {
    let root;
    act(() => {
      root = create(<LoadingCard />);
    });

    expect(root.toJSON()).toMatchSnapshot();
  });
});
