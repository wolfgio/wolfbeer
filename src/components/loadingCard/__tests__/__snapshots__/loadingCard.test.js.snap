// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingCard /> should render without error 1`] = `
<div
  className="sc-AykKC fJIIWK"
>
  <span>
    <span
      className="css-19bon8n react-loading-skeleton"
      style={
        Object {
          "animation": "animation-16jpnkj 1.2s ease-in-out infinite",
          "height": 280,
        }
      }
    >
      â€Œ
    </span>
  </span>
</div>
`;
